test/fail/access-self-prematurely.mo

```diff
@@ -0,0 +1,6 @@
+actor /*class () =*/ Self {
+    func caller(_callee : shared () -> async ()) { };
+    caller(Self.method1); // FIXME: should be error too
```

Comment:
The AST interpreter crashes her with
> execution error, Actor "ID:1" has no method "method1"
This could be resolved by allowing `(<actor>, <method>)` as an `async`-callable in the interpreters and make `DotE` produce those when projected out of an `actor`. (Idea by @crusso).

Reply:
I think that would also eliminate the need for the `increment` function...

Reply:
I was thinking of stashing the pairs into the `env.actors_env[actor_id]` where the `is_fullfilled` would give back `false`, but you may well be right. That would mean complicating the `DotE` evaluation a bit, though.